3-UsingAPIs/Week2/homework/ex4-diceRace.js

```diff
@@ -3,29 +3,34 @@
 Full description at: https://github.com/HackYourFuture/Homework/blob/main/3-UsingAPIs/Week2/README.md#exercise-4-dice-race
 
 1. Complete the function `rollTheDices()` by using `.map()` on the `dices` array 
-   to create an array of promises for use with `Promise.race()`.
+  to create an array of promises for use with `Promise.race()`.
 2. Refactor the function `main()` using async/await and try/catch.
 3. Once you got this working, you may observe that some dices continue rolling 
-   for some undetermined time after the promise returned by `Promise.race()` 
-   resolves. Do you know why? Add your answer as a comment to the bottom of the 
-   file.
+  for some undetermined time after the promise returned by `Promise.race()` 
+  resolves. Do you know why? Add your answer as a comment to the bottom of the 
+  file.
 ------------------------------------------------------------------------------*/
 // ! Do not remove this line
 const rollDice = require('../../helpers/pokerDiceRoller');
 
 function rollTheDices() {
   const dices = [1, 2, 3, 4, 5];
-  // TODO complete this function; use Promise.race() and rollDice()
+  const allDices = dices.map((dice) => rollDice(dice));
```

Comment:
```suggestion
  const allDices = dices.map((rollDice);
```

This would also work.